OpenXmlPowerTools/SpreadsheetDocumentManager.cs

```diff
@@ -146,12 +141,12 @@ private static List<string> GetHeaderReferences(string sheetName, string headerC
         /// <returns></returns>
         private static List<string> GetValueReferences(string sheetName, string headerColumn, List<string> headerList, List<string> colsToChart, string[][] valueTable)
         {
-            List<string> valueReferenceList = new List<string>();
-            int numRows = valueTable.GetLength(0);
+            var valueReferenceList = new List<string>();
+            var numRows = valueTable.GetLength(0);
 
-            foreach (string column in colsToChart)
+            foreach (var column in colsToChart)
             {
-                int dataColumn = headerList.IndexOf(column.ToUpper()) + 1;
+                var dataColumn = headerList.IndexOf(column.ToUpper()) + 1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=5308324)